.travis.yml

```diff
@@ -1,22 +1,52 @@
 language: cpp
+
+os:
+  - linux
+  - osx
+
+compiler:
+  - gcc
+  - clang
+
+branches:
+  only:
+    - master
+    - travis-osx
```

Comment:
Sure. I personally prefer to only build (and get notifications) regarding the most stable branch, but it depends a bit on which kind of project it is.